### PR TITLE
chore (v9 vr-tests): Remove usage of Screener

### PR DESCRIPTION
**Pull request by TristanWatanabe**
## Changes:
- removes `Screener` specific configs.
- removes `Screener` related dependencies from package.json.
- Replaces `Screener` tags with `StoryWright` tags,
- Replaces `screener` related imports with `storywright`.
## Related Issue(s)

* Follows #25977
* Part of #25078

### CODE REVIEW

**Comment by fabricteam** [comment]
## 📊 Bundle size report

<sub>🤖 This report was generated against <a href='https://github.com/microsoft/fluentui/commit/baf93e56255ce8136a965a7304d7d5bcc0042384'>baf93e56255ce8136a965a7304d7d5bcc0042384</a></sub>
<div id="bundleSizeComment9423"/>

**Comment by fabricteam** [comment]
## 🕵 fluentuiv9 No visual regressions between this PR and main
<div id="vrtCommentfluentuiv9"/>

File: apps/vr-tests-react-components/package.json [hunk 1]
```diff
@@ -5,15 +5,14 @@
   "description": "Visual regression tests for @fluentui/react-components",
   "scripts": {
     "build": "build-storybook -o dist/storybook",
-    "test:component": "storywright  --browsers chromium --url dist/storybook --destpath dist/screenshots --waitTimeScreenshot 500 --concurrency 4 --headless true",
     "convert": "bash convertStoriesToStoryWright.sh",
     "clean": "just-scripts clean",
     "format": "prettier . -w --ignore-path ../../.prettierignore",
     "lint": "just-scripts lint",
-    "screener": "just-scripts screener",
     "screener:build": "yarn build",
```
Comment (TristanWatanabe) [inline apps/vr-tests-react-components/package.json:12]: `screener:build` will be renamed in a separate PR along with other miscellaneous changes

**Review by micahgodbolt — APPROVED** [review]

**Review by ling1726 — APPROVED** [review]
